### PR TITLE
Custom spin (n7s-spin) added

### PR DESCRIPTION
**Pull request by rphh**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -103,7 +103,13 @@ RUN git clone --branch v4.0 --depth 1 https://github.com/cpputest/cpputest.git \
 # Download RTEMS
 RUN wget -q https://rtems-qual.io.esa.int/public_release/rtems-6-sparc-gr712rc-smp-4.tar.xz \
     && tar -xf rtems-6-sparc-gr712rc-smp-4.tar.xz -C /opt \
-    && rm -f rtems-6-sparc-gr712rc-smp-4.tar.xz
+	&& rm -f rtems-6-sparc-gr712rc-smp-4.tar.xz
```
Comment (Lurkerpas) [inline Dockerfile]: Why the change in indentation?

Comment (rphh) [reply Dockerfile]: fixed

**Review by Lurkerpas — APPROVED** [review]
